web/models/sitemaps.ex

```diff
@@ -0,0 +1,9 @@
+defmodule HexWeb.Sitemaps do
```

Comment:
this module could contain all logic related to sitemaps, so I'm thinking about moving all logic here from:
- https://github.com/hexpm/hex_web/blob/9ca602f77010c4ca5883f566b757c978b1eb59d1/web/models/package.ex#L62:L73
- https://github.com/hexpm/hex_web/blob/9ca602f77010c4ca5883f566b757c978b1eb59d1/web/controllers/sitemap_controller.ex#L5:L6